index.js

```diff
@@ -207,14 +207,20 @@ async function printContactenation(files) {
 }
 
 async function getTruffleRoot() {
-  try {
-    const truffleConfiPath = await findUp("truffle.js");
-    return getDirPath(truffleConfiPath);
-  } catch (error) {
-    throw new Error(
-      "Truffle Flattener must be run inside a Truffle project: truffle.js not found"
-    );
-  }
+    try {
+        const truffleConfiPath = await findUp("truffle.js");
+        return getDirPath(truffleConfiPath);
+    } catch (error) {
+        try {
+            const truffleConfiPath = await findUp("truffle-config.js");
+            return getDirPath(truffleConfiPath);
+        }
+        catch (error) {
+            throw new Error(
+                "Truffle Flattener must be run inside a Truffle project: truffle.js not found"
```

Comment:
This message is outdated

Reply:
Can you run `prettier` to fix the formatting? It's available as a dev dependency in this project.